### PR TITLE
made sure that brage embargo dates always ends up with start-of-day t…

### PR DESCRIPTION
**Pull request by ketilaa**
…imestamps in NVA

### CODE REVIEW

**Review by anetteOlli — APPROVED** [review]

File: src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java [hunk 1]
```diff
@@ -112,10 +112,12 @@ var record = new Record();
...
         var updatedRecord = EmbargoParser.checkForEmbargoFromSuppliedEmbargoFile(record, embargos);
         assertThat(appender.getMessages(), not(containsString("Embargo file not found: ")));
-        updatedRecord.getContentBundle()
-            .getContentFiles()
-            .forEach(actualContentFile ->
-                         assertThat(actualContentFile.getEmbargoDate(), is(notNullValue())));
+
+        assertThat(updatedRecord.getContentBundle().getContentFiles(), hasSize(2));
+        assertThat(updatedRecord.getContentBundle().getContentFiles().get(0).getEmbargoDate(),
+                   is(equalTo(Instant.parse("2023-09-30T22:00:00Z"))));
```
Comment (anetteOlli) [inline src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java:118]: Gjerne også en test for vinter-tid? Siden det gir annen zulu-tid.

**Review by anetteOlli — CHANGES_REQUESTED** [review]

File: src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java (hunk 1 above)
Comment (ketilaa) [reply src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java:120]: Added a test to cover daylight saving when setting embargo date

File: src/main/java/no/sikt/nva/model/Embargo.java [hunk 2]
```diff
@@ -89,8 +93,8 @@ public void setFilename(String filename) {
     private Instant perseFiveYearDateToInstant() {
         var fiveDigitYear = DateTimeFormatter.ofPattern("yyyyy-MM-dd");
         var dateTime = ZonedDateTime.of(LocalDate.parse(date, fiveDigitYear),
-                                LocalTime.now(),
-                                ZoneId.systemDefault());
+                                START_OF_DAY,
```
[the remark this answers is not in the document]
Comment (ketilaa) [reply src/main/java/no/sikt/nva/model/Embargo.java:96]: Check below

File: src/main/java/no/sikt/nva/model/Embargo.java [hunk 3]
```diff
@@ -11,6 +11,8 @@
 
 public class Embargo {
 
+    private static final ZoneId EUROPE_OSLO = ZoneId.of("Europe/Oslo");
```
[the remark this answers is not in the document]
Comment (ketilaa) [reply src/main/java/no/sikt/nva/model/Embargo.java:14]: Yes, check below

**Review by truhacevkir — APPROVED** [review]

**Pull request merged**